camptix.php

```diff
@@ -1929,6 +1953,414 @@ function get_currencies() {
 				'label' => __( 'South African Rand', 'camptix' ),
 				'format' => 'R %s',
 			),
+			'DZD'=> array(
```

Comment:
Ideally, I think, these additions would be alphabetized by their array key, so it would be easier to find them in the code.

Reply:
Will do that once format issue is taken care of.